### ISSUE TITLE
Does this tool re-download deleted podcast episodes?

### ISSUE THREAD

**Issue by WolfgangDpunkt** [lpar/podtools#2]
That sounds very promising.
Does this tool re-download deleted podcast episodes?

Before I try it myself, I'll just ask, possibly as a suggestion to add this info in the readme ;-)

For me, this is the most important feature, because I remove episodes after finished listening to it, and it would be fatal if they are then downloaded again.
Thank you very much!

**Comment by lpar** [comment]
Yes, it was designed for archiving all episodes, so (for example) if an old episode of This American Life is reposted to their feed, and you have deleted that episode, it'll be downloaded again. It doesn't keep a separate history of what it has downloaded.

**Comment by WolfgangDpunkt** [comment]
Thank you for this explanation, it was very helpful.
Have a nice day! :-)
